libnvme-sys/src/identify.rs

```diff
@@ -0,0 +1,700 @@
+use std::ffi::c_char;
```

Comment:
FYI: I didn't compare the stuff in this file against the spec

Reply:
We can follow up on this later if you want.  There's another commit here that starts an nvme spec crate and this definition should probably be moved there.